ros2_foxglove_bridge_sdk/include/foxglove_bridge/ros2_foxglove_bridge.hpp

```diff
@@ -67,7 +89,8 @@ class FoxgloveBridge : public rclcpp::Node {
   std::unordered_map<foxglove_ws::ChannelId, foxglove_ws::ChannelWithoutId> _advertisedTopics;
   std::unordered_map<foxglove_ws::ServiceId, foxglove_ws::ServiceWithoutId> _advertisedServices;
   std::unordered_map<foxglove_ws::ChannelId, SubscriptionsByClient> _subscriptions;
-  PublicationsByClient _clientAdvertisedTopics;
+  std::unordered_map<std::pair<uint32_t, uint32_t>, ClientAdvertisement, PairHash>
```

Comment:
what is stored inside the pair? It would be more readable if you would use the type aliases defined above

Reply:
ah, I guess this should be 
```suggestion
  std::unordered_map<ChannelAndClientId, ClientAdvertisement, PairHash>
```
?